package.json

```diff
@@ -31,11 +31,14 @@
     "hardhat-deploy-ethers": "^0.3.0-beta.7",
     "hardhat-gas-reporter": "^1.0.3",
     "hardhat-log-remover": "^2.0.0",
+    "prettier": "^2.2.1",
+    "prettier-plugin-solidity": "^1.0.0-beta.7",
```

Comment:
You would want these globally installed not locally here.

Reply:
why globally: https://github.com/prettier-solidity/prettier-plugin-solidity

doesn't it say to do locally here?

Reply:
installing it globally allows to use it with vscode on any project, not needing to install for every single project.

I don't even remember installing, when I installed the Solidity vscode extension is got installed globally as part of that.
https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity

jump to : Formatting using Prettier and the Prettier Solidity Plugin